src/screens/Learn/Landing/SectionButton.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { GreenButton, WhiteButton, StyledLink } from './SectionButton.style';
+
+export enum ButtonTheme {
+  WHITE,
+  GREEN,
+}
+
+const SectionButton = (props: {
+  cta: string;
+  redirect: string;
+  theme: ButtonTheme;
+}) => {
+  const themeToButtonMap = {
+    [ButtonTheme.WHITE]: WhiteButton,
+    [ButtonTheme.GREEN]: GreenButton,
+  };
+
+  const ButtonComponent = themeToButtonMap[props.theme];
+
+  return (
+    <StyledLink to={props.redirect}>
+      <ButtonComponent theme={props.theme} disableRipple>
```

Comment:
You can pass the `disableRipple` property on the styles, that way it doesn't have to be repeated each time

```tsx
const ButtonComponent = styled(MuiButton)
  .attrs(props => ({
    disableRipple: true
  })`
    color: blue;
  `;
```

and later

```tsx
{/* no ripple */}
<ButtonComponent  />
```

Reply:
`<ButtonComponent  disabledRipple/>` is just written once. the component that will be reused is :
```
            <SectionButton
              cta={section.buttonCta}
              redirect={section.buttonRedirect}
              theme={ButtonTheme.WHITE}
            />
```